### PR TITLE
New animations

### PR DESCRIPTION
**Pull request by Ryu-k**
Added few animations.
- Scaling Entrances.
- Fading Entrances "small" version added.

### CODE REVIEW

File: public/sources/attributes/animateDropdown/Component.js [hunk 1]
```diff
@@ -114,6 +114,33 @@ export default class AnimateDropdown extends Dropdown {
     //     ]
     //   }
     // },
+    {
```
Comment (nikitahl) [inline public/sources/attributes/animateDropdown/Component.js:117]: As I understand this PR ([2175](https://github.com/VisualComposer/builder/pull/2175/files)) can be dismissed, as these are the same changes in the same file.

File: public/sources/assetsLibrary/animate/src/animate.css [hunk 2]
```diff
@@ -1132,6 +1132,156 @@
...
+    -webkit-transform: translate3d(0, 0, 0) scaleX(1);
+            transform: translate3d(0, 0, 0) scaleX(1);
+  }
+}
+
+[data-vcv-o-animated="true"][data-vce-animate="vce-o-animate--scaleInLeft"] {
+    animation-name: vce-o-animate--scaleInLeft;
+}
+
+@-webkit-keyframes vce-o-animate--scaleInRight {
```
Comment (nikitahl) [inline public/sources/assetsLibrary/animate/src/animate.css:1255]: scaleInRight has the same rules as scaleInLeft

File: public/sources/assetsLibrary/animate/src/animate.css [hunk 3]
```diff
@@ -1132,6 +1132,156 @@
...
+            transform: translate3d(-50px, 0, 0) scaleX(1.1);
+  }
+  57% {
+    opacity: 1;
+    -webkit-transform: translate3d(0, 0, 0) scaleX(1);
+            transform: translate3d(0, 0, 0) scaleX(1);
+  }
+}
+
+@keyframes vce-o-animate--scaleInRight {
```
Comment (nikitahl) [inline public/sources/assetsLibrary/animate/src/animate.css:1268]: scaleInRight has the same rules as scaleInLeft

**Review by nikitahl — CHANGES_REQUESTED** [review]

**Review by Ryu-k — COMMENTED** [review]
changed -50 to 50 in the scaleInRight animation!

**Comment by nikitahl** [comment]
> changed -50 to 50 in the scaleInRight animation!

Inside animate.bundle.js the `@keyframes vce-o-animate--scaleInRight` still has -50%. Need to re-build the bundle.

File: public/sources/assetsLibrary/animate/dist/animate.bundle.css [hunk 4]
```diff
@@ -1 +1 @@
...
...ollOut-xl]{animation-name:vce-o-animate--rollOut}[data-vcv-o-animated=true][data-vce-animate*=vce-o-animate--zoomIn-xl]{animation-name:vce-o-animate--zoomIn}[data-vcv-o-animated=true][data-vce-animate*=vce-o-animate--zoomInDown-xl]{animation-name:vce-o-animate--zoomInDown}[data-vcv-o-animated=true][data-vce-animate*=vce-o-animate--zoomInLeft-xl]{animation-name:vce-o-animate--zoomInLeft}[data-vcv-o-animated=true][data-vce-animate*=vce-o-animate--zoomInRight-xl]{animation-name:vce-o-animate--zoomInRight}[data-vcv-o-animated=true][data-vce-animate*=vce-o-animate--zoomInUp-xl]{animation-name:vce-o-animate--zoomInUp}[data-vcv-o-animated=true][data-vce-animate*=vce-o-animate--zoomOut-xl]{animation-name:vce-o-animate--zoomOut}[data-vcv-o-animated=true][data-vce-animate*=vce-o-animate--zoomOutDown-xl]{animation-name:vce-o-animate--zoomOutDown}[data-vcv-o-animated=true][data-vce-animate*=vce-o-animate--zoomOutLeft-xl]{animation-name:vce-o-animate--zoomOutLeft}[data-vcv-o-animated=true][data-vce-animate*=vce-o-animate--zoomOutRight-xl]{animation-name:vce-o-animate--zoomOutRight}[data-vcv-o-animated=true][data-vce-animate*=vce-o-animate--zoomOutUp-xl]{animation-name:vce-o-animate--zoomOutUp}}
```
Comment (nikitahl) [inline public/sources/assetsLibrary/animate/dist/animate.bundle.css:1]: Inside animate.bundle.js the @keyframes vce-o-animate--scaleInRight still has -50%. Need to re-build the bundle.

**Review by nikitahl — CHANGES_REQUESTED** [review]

**Comment by Ryu-k** [comment]
should be ok now, sry for the delay

**Review by nikitahl — APPROVED** [review]

**Pull request merged**